docs/HowTo/Get-Started/Starting-node.md

```diff
@@ -147,6 +148,14 @@ besu --network=goerli --data-path=<path>/<goerlidata-path>
 
 Where `<path>` and `<goerlidata-path>` are the path and directory to save the Goerli chain data to.
 
+## Run a node on the Merge Testnet
```

Comment:
Sorry, my earlier feedback was probably misleading. I think it's fine to use the name Kiln in this heading.

Although Testnet should probably be lowercase here.